cmd/release-controller/sync.go

```diff
@@ -299,10 +299,23 @@ func (c *Controller) syncPending(release *releasecontroller.Release, pendingTags
 			rewriteValue := tag.Annotations[releasecontroller.ReleaseAnnotationRewrite]
 			if len(rewriteValue) == 0 {
 				klog.V(2).Infof("Rewriting pending release %s", tag.Name)
-				isi, err := c.imageClient.ImageStreamImages(release.Source.Namespace).Get(context.TODO(), fmt.Sprintf("%s@%s", release.Source.Name, id), metav1.GetOptions{})
+				var isi *imagev1.ImageStreamImage
+				isi, err = c.imageClient.ImageStreamImages(release.Source.Namespace).Get(context.TODO(), fmt.Sprintf("%s@%s", release.Source.Name, id), metav1.GetOptions{})
```

Comment:
This could be kept as a single line with `:=`.

Reply:
That changes the ability to reassign the `isi` value down below and continue using it further down in the code.

Reply:
Ignore me!  I don't know what I'm talking about

Reply:
Corrected!